scout/build/managed_variant.py

```diff
@@ -32,7 +35,15 @@ def build_managed_variant(managed_variant_info):
             description=managed_variant_info.get("description", ""),
         )
     except KeyError:
-        raise KeyError("Managed variant has to have chr, pos, ref and alt.")
+        flash(
+            "Managed variant has to have chr, pos, ref and alt",
+            "danger",
+        )
+    except ValueError as ve:
```

Comment:
Tested again after introducing this change: works still!

<img width="712" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/04005a08-e3c8-4d83-a43c-d216e33aa970">